db.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
- React App and Json db are both running on port 3000, I would change that 
- Login In functionality needs to be implemented
- When I add more than one of the same item to the cart and try and remove one, it removes all of the same item from the cart
- Inventory needs to be updated when adding an item to the cart